### ISSUE TITLE
[Android] Jurassic World Evolution crash on 1x Resolution

### ISSUE THREAD

**Issue by zZardd** [yuzu-emu/yuzu#12917]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

android-228 - android-230

### Description of Issue

If you open the game at 1x res, it will immediately crash on version 228 and the latest, 230. Which is not happening in 227. 
Here is a clip from 230:

https://github.com/yuzu-emu/yuzu/assets/108114549/1958fa35-0c47-4fa7-9793-bcfe425d5917

### Expected Behavior

Well, it supposed to work since it was working on 227 at 1x resolution as well.
Here's a clip from 227:

https://github.com/yuzu-emu/yuzu/assets/108114549/4034611a-b4a3-4709-aa4a-de8a4c46854e

### Reproduction Steps

Save file: 
[010050A011344000.zip](https://github.com/yuzu-emu/yuzu/files/14160098/010050A011344000.zip)

### Log File

[yuzu_log.txt.old (1).txt](https://github.com/yuzu-emu/yuzu/files/14160109/yuzu_log.txt.old.1.txt)

### System Configuration

CPU: Snapdragon 778g
GPU/Driver: Adreno 642L
RAM: 8GB
OS: Android 14

**Comment by liamwhite** [comment]
Is this intending to say that it works with 2x?

**Comment by zZardd** [comment]
> Is this intending to say that it works with 2x?

No. It only works up to 1x res in my device..

**Comment by liamwhite** [comment]
Does this mean it works at 0.5x?

**Comment by zZardd** [comment]
> Does this mean it works at 0.5x?

Yes, but on my experience it's much playable on 1x since the buildings doesn't render in 0.5x and in 0.75x res..
